dbt_subprojects/daily_spellbook/models/_projects/gmx/event/avalanche_c/gmx_v2_avalanche_c_oracle_price_update.sql

```diff
@@ -0,0 +1,149 @@
+{{
+  config(
+    schema = 'gmx_v2_avalanche_c',
+    alias = 'oracle_price_update',
+    materialized = 'table'
+    )
+}}
+
+{%- set event_name = 'OraclePriceUpdate' -%}
+{%- set blockchain_name = 'avalanche_c' -%}
+
+
+WITH evt_data_1 AS (
+    SELECT 
+        -- Main Variables
+        '{{ blockchain_name }}' AS blockchain,
+        evt_block_time AS block_time,
+        evt_block_number AS block_number, 
+        evt_tx_hash AS tx_hash,
+        evt_index AS index,
+        contract_address,
+        eventName AS event_name,
+        eventData AS data,
+        msgSender AS msg_sender
+    FROM {{ source('gmx_v2_avalanche_c','EventEmitter_evt_EventLog1')}}
+    WHERE eventName = '{{ event_name }}'
+        AND evt_block_time > DATE '2023-08-01'
+    ORDER BY evt_block_time ASC
```

Comment:
i.e.
```suggestion
    WHERE eventName = '{{ event_name }}'
        {% if is_incremental() %}
        AND {{ incremental_predicate('evt_block_time') }}
        {% else %}
        AND evt_block_time > DATE '2023-08-01'
        {% endif %}
```

We may remove `ORDER BY` in the CTE as there is `UNION` operation after.

Reply:
Thank you for your comment. I will do it. It's really worth to make it incremental because that table is large.